var/ramble/repos/builtin/modifiers/nccl-env/modifier.py

```diff
@@ -0,0 +1,714 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.modkit import *  # noqa: F403
```

Comment:
nit: the noqa comment should not be needed.

Reply:
Thanks, fixed.